control-plane-operator/hostedclusterconfigoperator/controllers/resources/konnectivity/params.go

```diff
@@ -56,6 +56,24 @@ func NewKonnectivityParams(hcp *hyperv1.HostedControlPlane, images map[string]st
 			SuccessThreshold:    1,
 		},
 	}
+
+	p.DeploymentConfig.ReadinessProbes = config.ReadinessProbes{
+		konnectivityAgentContainer().Name: {
+			ProbeHandler: corev1.ProbeHandler{
+				HTTPGet: &corev1.HTTPGetAction{
+					Scheme: corev1.URISchemeHTTP,
+					Port:   intstr.FromInt(int(healthPort)),
+					Path:   "readyz",
+				},
+			},
+			InitialDelaySeconds: 120,
```

Comment:
See internal review comments that I provided with respect to speeding up the readiness probe intervals so as to not significantly delay rollouts and readiness recovery.

Reply:
thanks, i implemented the values here to align with the internal comments.